modules/installation-installing-bare-metal.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+
+[id="installation-installing-bare-metal-{context}"]
+= Creating the cluster
+
+To create the {product-title} cluster, you provision machines by using the Ingition config
+files that you generated with the installation program.
+
+.Prerequisites
+
+* Create the required infrastructure for the cluster.
+* You obtained the installation program and generated the Ignition config files
+for your cluster.
+* You used the Ignition config files to create {op-system} machines for your
+cluster.
+* Your machines have internet access.
+
+
+.Procedure
+
+. Start and monitor the installation process:
++
+----
+$ openshift-install --dir <installation_directory> wait-for bootstrap-complete
+INFO Waiting up to 30m0s for the Kubernetes API at https://api.test.example.com:6443...
+INFO API v1.12.4+c53f462 up
+INFO Waiting up to 30m0s for the bootstrap-complete event...
+----
++
+The command succeeds when the Kubernetes APIServer signals that it has been
+bootstrapped on the control plane machines.
+
+. Remove the bootstrap machine from the load balancer.
```

Comment:
We should include text here (or as a subsequent step) that informs the user that they can destroy the bootstrap VM at this point.